.github/workflows/generate-token-lists.yml

```diff
@@ -116,10 +116,23 @@ jobs:
               - ArbTokenLists/421613_arbed_full.json
             version: false
             command: yarn fullList --l2NetworkID 421613 --newArbifiedList ./src/ArbTokenLists/421613_arbed_full.json --skipValidation
+
+          - name: ArbSepolia Arbify Uniswap
+            paths:
+              - ArbTokenLists/421614_arbed_coinmarketcap.json
+            version: true
+            command: yarn arbify --l2NetworkID 421614  --ignorePreviousList 421614_arbed_uniswap_labs_default.json --tokenList https://gateway.ipfs.io/ipns/tokens.uniswap.org --newArbifiedList ./src/ArbTokenLists/421614_arbed_uniswap_labs_default.json
```

Comment:
`--ignorePreviousList` is temporary because we don't have an existing list yet and the arbifying process won't work without this flag

Reply:
Reverted, list was uploaded to s3